<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRYPTOFICHIER</class>
 <widget class="QMainWindow" name="CRYPTOFICHIER">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CRYPTOFICHIER</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 52, 52);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>681</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>C:/Users/ML/.designer/backup/image/32CYPHER.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_35">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 255, 0);
background:transparent;
font: 75 16pt &quot;Century Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>Cryptage </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_68">
       <property name="styleSheet">
        <string notr="true">color: #ffffff;
background:transparent;
font: 75 16pt &quot;Century Gothic&quot;;
</string>
       </property>
       <property name="text">
        <string>Fichier Word .docx</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="decrypter_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;5by7&quot;;
color: rgb(255, 255, 255);
background-color: rgb(6, 100, 135);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Decrypter</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="crypter_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(6, 100, 135);
font: 14pt &quot;5by7&quot;;
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);font: 14pt &quot;5by7&quot;;
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Crypter</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="msg_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(0, 170, 0,0.7);

color: rgb(255, 255, 255);
border-radius:10px;
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string> Le fichier a été crypté avec succes</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255,0,0,0.8);
color: rgb(255, 255, 255);
border-radius:10px;
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string> Entrer le chemin d'abord </string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255,0,0,0.8);
color: rgb(255, 255, 255);
border-radius:10px;
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string> le fichier n'existe pas ou type incorrect</string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_4">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgba(0, 170, 0,0.7);

color: rgb(255, 255, 255);
border-radius:10px;
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string> Le fichier a été decrypté avec succes</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="chemin">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">

color:#ffffff;
border-radius: 0px;
border:1px solid #9cd3e2;
background-color: rgb(33, 65, 103);
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>999999999</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true">chemin de fichier à crypter / décrypter</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="parcourire_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;5by7&quot;;
color: rgb(255, 255, 255);
background-color: rgb(6, 100, 135);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Parcourire</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: 14pt &quot;5by7&quot;;

font: 14pt &quot;Century Gothic&quot;;

color:#ffffff;</string>
    </property>
    <property name="text">
     <string> Auto générer les fichier .pem des cléfs</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: 14pt &quot;5by7&quot;;

font: 14pt &quot;Century Gothic&quot;;

color:#ffffff;</string>
    </property>
    <property name="text">
     <string>Chemin Du fichier à crypter/ decrypter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmer_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;5by7&quot;;
color: rgb(255, 255, 255);
background-color: rgb(6, 100, 135);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Au-Gen Cléfs</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="rchemin">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">

color:#ffffff;
border-radius: 0px;
border:1px solid #9cd3e2;
background-color: rgb(33, 65, 103);
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>999999999</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true">chemin de fichier resultant </string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font: 14pt &quot;5by7&quot;;

font: 14pt &quot;Century Gothic&quot;;

color:#ffffff;</string>
    </property>
    <property name="text">
     <string>Chemin Du fichier resultant </string>
    </property>
   </widget>
   <widget class="QLabel" name="msg_5">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>270</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255,0,0,0.8);
color: rgb(255, 255, 255);
border-radius:10px;
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>Indiquer chemin fichier resultant </string>
    </property>
   </widget>
   <widget class="QPushButton" name="parcourire_btn_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(6, 100, 135);
font: 14pt &quot;5by7&quot;;
color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Parcourire</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="prvkey">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>80</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;5by7&quot;;
color: rgb(255, 255, 255);
background-color: rgb(6, 100, 135);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Fichier Clef privé</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pubkey">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;5by7&quot;;
color: rgb(255, 255, 255);
background-color: rgb(6, 100, 135);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>Fichier Clef public</string>
    </property>
    <property name="shortcut">
     <string>Return</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="chemin_prvkey">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">

color:#ffffff;
border-radius: 0px;
border:1px solid #9cd3e2;
background-color: rgb(33, 65, 103);
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>999999999</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true">chemin de la clé privé  .pem</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="chemin_pubkey">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>15</width>
      <height>15</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">

color:#ffffff;
border-radius: 0px;
border:1px solid #9cd3e2;
background-color: rgb(33, 65, 103);
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="inputMask">
     <string notr="true"/>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="maxLength">
     <number>999999999</number>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string notr="true">chemin de la clé public  .pem</string>
    </property>
    <property name="cursorMoveStyle">
     <enum>Qt::LogicalMoveStyle</enum>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="msg_6">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>160</y>
      <width>311</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(255,0,0,0.8);
color: rgb(255, 255, 255);
border-radius:10px;
font: 12pt &quot;Century Gothic&quot;;</string>
    </property>
    <property name="text">
     <string>le fichier des clef doit être /fichier.pem</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
